### PR TITLE
Changed page structure

### PR DESCRIPTION
**Pull request by ajantsch**
unify healthy-together.eu and gesund-zusammen.de

### CODE REVIEW

File: src/components/common/InfoBox.tsx [hunk 1]
```diff
@@ -33,21 +37,24 @@ const StyledTitle: AnyStyledComponent = styled(Typography)`
...
+    text-align: center;
+    padding: 1.5rem 2rem 2rem;
 
     &:last-child {
       padding-bottom: 2rem;
     }
   }
 `;
 
+const SquaredImageWrapper: AnyStyledComponent = styled.div`
```
Comment (manzke) [inline src/components/common/InfoBox.tsx:56]: can we limit the size of the illus a little bit? quite big right now

File: src/components/common/PartnerItems.tsx [hunk 2]
```diff
@@ -52,51 +52,46 @@ class PartnerItems extends React.Component<IPartnerItemsProps, {}> {
   render = () => {
     return (
       <Box id="partners" paddingBottom={4} marginTop={4}>
-        <Fade right cascade>
```
Comment (manzke) [inline src/components/common/PartnerItems.tsx:55]: 👎 dislike ;)

File: src/components/content/Accelerator.tsx [hunk 3]
```diff
@@ -31,32 +37,76 @@ class Accelerator extends React.PureComponent<WithTranslation, {}> {
...
+        </Box>
+
+        <Box paddingBottom={4} marginTop={4}>
+          <LinkButton
+            color="primary"
+            variant="contained"
+            disableFocusRipple={true}
+            href={`/${this.props.i18n.language}/program`}
+          >
+            Go to the program
```
Comment (manzke) [inline src/components/content/Accelerator.tsx]: translation?

File: src/components/content/Start.tsx [hunk 4]
```diff
@@ -1,17 +1,23 @@
...
 import ContentEN from "../../data/landing/landing_en.md";
 import ContentFR from "../../data/landing/landing_fr.md";
 import ContentIT from "../../data/landing/landing_it.md";
 import ContentES from "../../data/landing/landing_es.md";
 
+import IlluAccelerator from "../../images/illu_accelerator.svg";
+import IlluApp from "../../images/illu_app.svg";
+import IlluInitiative from "../../images/illu_initiative.svg";
+
 class Landing extends React.PureComponent<WithTranslation, {}> {
```
Comment (manzke) [inline src/components/content/Start.tsx:21]: should we change it to start?

File: src/components/layouts/Initiative/Header.tsx [hunk 5]
```diff
@@ -70,7 +70,6 @@ const ApplyButton: AnyStyledComponent = styled(Button)`
     padding-left: 2rem;
 
     @media (min-width: 600px) {
```
Comment (manzke) [inline src/components/layouts/Initiative/Header.tsx:72]: delete if nothing in it

File: src/components/layouts/Shared/HeaderNav.tsx [hunk 6]
```diff
@@ -108,15 +118,40 @@ const headerNavItemStyles = css`
...
+
+  @media (min-width: 600px) {
+    display: inline-block;
+    width: auto;
   }
 `;
 
-const HeaderNavLinkItem: AnyStyledComponent = styled(Link)`
-  ${headerNavItemStyles}
+const HeaderNavItemWithSubitems: AnyStyledComponent = styled.div`
```
Comment (manzke) [inline src/components/layouts/Shared/HeaderNav.tsx:158]: please check alignment. they should be aligned with the parent

File: src/locales/de.json [hunk 7]
```diff
@@ -9,9 +9,25 @@
   "header": {
     "nav": {
       "initiative": "Initiative",
-      "initiatives": "Datenbank",
-      "supporters": "Partner",
-      "faqs": "Fragen?"
+      "database": "Datenbank",
+      "accelerator": "Accelerator",
+      "partners": "Partner",
```
Comment (manzke) [inline src/locales/de.json:14]: still needed?

File: src/locales/de.json [hunk 8]
```diff
@@ -9,9 +9,25 @@
...
+      "program": "Programm"
+    }
+  },
+  "start": {
+    "box1": {
+      "title": "Die Initiative",
+      "content": "Lesen Sie mehr über unsere Initative."
+    },
+    "box2": {
+      "title": "EINSgegenCorona",
```
Comment (manzke) [inline src/locales/de.json]: coming soon

File: src/components/content/Start.tsx [hunk 9]
```diff
@@ -35,28 +41,58 @@ class Landing extends React.PureComponent<WithTranslation, {}> {
...
+                <InfoBox
+                  title={this.props.t("start.box1.title")}
+                  content={this.props.t("start.box1.content")}
+                  illustration={IlluInitiative}
+                />
+              </StyledLink>
+            </Grid>
+            <Grid item xs={12} sm={6} md={4} xl={3}>
+              <ExternalLink
+                href={`https://ichhabs.app`}
```
Comment (manzke) [inline src/components/content/Start.tsx:59]: remove for now

File: src/components/content/Start.tsx [hunk 10]
```diff
@@ -35,28 +41,58 @@ class Landing extends React.PureComponent<WithTranslation, {}> {
...
-              <InfoBox
-                title={this.props.t("initiative.box3.title")}
-                content={this.props.t("initiative.box3.content")}
-              />
+        </Box>
+
+        <Box paddingBottom={4} marginTop={4}>
+          <Grid container spacing={4}>
+            <Grid item xs={12} sm={6} md={4} xl={3}>
+              <StyledLink to={`/${this.props.i18n.language}/initiative`}>
```
Comment (manzke) [inline src/components/content/Start.tsx:49]: should we externalise the links into the json files?

File: src/components/content/Accelerator.tsx (hunk 3 above)
Comment (ajantsch) [reply src/components/content/Accelerator.tsx]: already in

File: src/components/layouts/Shared/HeaderNav.tsx (hunk 6 above)
Comment (ajantsch) [reply src/components/layouts/Shared/HeaderNav.tsx:158]: left aligned? because they are right-aligned right now

Comment (manzke) [reply src/components/layouts/Shared/HeaderNav.tsx:158]: Apr-16-2020 16-05-05

yeah let's do left for now

File: src/components/content/Start.tsx (hunk 10 above)
Comment (ajantsch) [reply src/components/content/Start.tsx:49]: Unless we would introduce localized routes (e.g. `/database` in en and `/datenbank` in de) I would not handle it in the localization files

**Pull request merged**